WebDevStudios/wd_s
# Bumps to Tailwind 3.0.24

Closes #804

### DESCRIPTION
Bumps to the latest Tailwind and brings over the additional updates noted in #804 to move us out of Tailwind 2.0.

### SCREENSHOTS
<img width="593" alt="tailwind config updates" src="https://user-images.githubusercontent.com/954724/167640518-5aa36701-8163-4f6e-b0f7-b750a2fcfc63.png">
<img width="844" alt="package.json file updates" src="https://user-images.githubusercontent.com/954724/167640634-bc333af8-2125-4c52-8009-545ff4b28749.png">

Sourcemaps:
<img width="1121" alt="sourcemaps displayed in local install" src="https://user-images.githubusercontent.com/954724/167640746-fe241199-47f4-46de-9ef2-fa682bb46064.png">

### OTHER


### STEPS TO VERIFY
- Check out
- `npm i`
- Run `start`, `watch`, `build`, `lint`, and `format` while making updates to ensure the frontend is updated and continues working

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

I'll need to make a couple of changes to our wiki page on Tailwind config, but otherwise things continue to operate as usual without any changes in workflow.
